auslib/AUS.py

```diff
@@ -66,6 +65,14 @@ def evaluateRules(self, updateQuery):
         rule = rules[0]
         self.log.debug("Matching rule: %s" % rule)
 
+        # If the rule requires a whitelisted imei, check via our whitelist blob
+        if rule['whitelist']:
+            self.log.debug("Matching rule requires whitelisted IMEI")
```

Comment:
Nit: Don't mention IMEI here - that aspect is encapsulated within the whitelist blob. Out here it's just a generic whitelisting column.